README.md

```diff
@@ -100,8 +100,7 @@ some issues cannot be fixed properly.**
 This option will enable caching of the linting results into a file.
 This is particularly useful in reducing linting time when doing a full build.
 
-The cache file is written to the `./node_modules/.cache` directory, thanks to the usage
-of the [find-cache-dir](https://www.npmjs.com/package/find-cache-dir) module.
+The cache file is written to the `./node_modules/.cache` directory.
```

Comment:
My idea is: since we use a module that handle this, we don't really know what the deps will rely on, so maybe we should say "we use X" with a link so people can have an up to date link? I am just afraid this doc become out of date someday :)